mlir/test/Dialect/Linalg/transform-lower-pack.mlir

```diff
@@ -62,14 +62,15 @@ module attributes {transform.with_named_sequence} {
 // -----
 
 // CHECK-LABEL: func.func @pack_as_pad(
+// CHECK: %[[SRC:.+]]: tensor<129x47x16x16xf32>,
+// CHECK: %[[OUT:.+]]: tensor<1x1x1x1x136x64x16x16xf32>)
```

Comment:
Tip: you can write `%[[SRC:[a-zA-Z0-9]+]]` to avoid capturing the types, which is not important in the lit tests.